### PR TITLE
WT-6856 Test configuration can be retrieved from file + Logging activation

### PR DESCRIPTION
**Pull request by etienneptl**

### CODE REVIEW

File: test/cppsuite/tests/poc.cxx [hunk 1]
```diff
@@ -53,41 +54,112 @@ const std::string poc_test::test::default_config =
...
+        std::cout << "Trace level\t:" << trace_level << std::endl;
 
-    poc_test(cfg, trace_level).run();
-    return (0);
-}
+        poc_test(cfg, trace_level).run();
+    }
+
+    return (error_code);
+}
```
Comment (etienneptl) [inline test/cppsuite/tests/poc.cxx:165]: Will be fixed

File: test/cppsuite/configurations/config.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+collection_count=2
+key_count=5
+value_size=10
+read_threads=1
+duration_seconds=10
+cache_size_mb=1000
+stat_cache_size=
+START_SUBCONFIG
+START_SUBCONFIG
```
Comment (etienneptl) [inline test/cppsuite/configurations/config.txt:9]: To remove

File: test/cppsuite/configurations/config.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
+collection_count=2
+key_count=5
+value_size=10
+read_threads=1
+duration_seconds=10
+cache_size_mb=1000
+stat_cache_size=
+START_SUBCONFIG
```
Comment (luke-pearson) [inline test/cppsuite/configurations/config.txt:8]: Use '{' and '}' instead of these labels.

File: test/cppsuite/configurations/config.txt [hunk 4]
```diff
@@ -0,0 +1,19 @@
+collection_count=2
```
Comment (luke-pearson) [inline test/cppsuite/configurations/config.txt:1]: Rename this file to config_poc_short.txt

Also add a comment line explaining what this config is and what it does in terms of the workload at the top of this file.

File: test/cppsuite/test_harness/test.h [hunk 5]
```diff
@@ -97,13 +97,20 @@ class test {
...
+        bool enable_logging, enable_tracking = false, is_success = true;
+        std::string enable_logging_str;
 
         /* Build the database creation config string. */
         std::string db_create_config = CONNECTION_CREATE;
 
         testutil_check(_configuration->get_int(CACHE_SIZE_MB, cache_size_mb));
         db_create_config += ",statistics=(fast),cache_size=" + std::to_string(cache_size_mb) + "MB";
+        testutil_check(_configuration->get_bool(ENABLE_LOGGING, enable_logging));
+        if (enable_logging)
```
Comment (luke-pearson) [inline test/cppsuite/test_harness/test.h:109]: Lets just do:
```
        db_create_config += ",log=(enabled=" + enable_logging ? "true" : "false" + ")";
```
or 
```
        if (enable_logging)
           db_create_config += ",log=(enabled=true)";
        else
           db_create_config += ",log=(enabled=false)";
```

**Review by luke-pearson — APPROVED** [review]
LGTM.

**Pull request merged**